src/Evolution/Systems/GeneralizedHarmonic/Constraints.hpp

```diff
@@ -300,14 +305,163 @@ void f_constraint(
+ *      & + & \Lambda^2\bigl( C_a C_b
+ *      +g^{ij}m^{cd} C_{iac} C_{jbd}\bigr)
+ * \Bigr]\nonumber\\
+ * &=&
+ *      m^{ab} F_a F_b
+ *      +m^{ab}g^{ij} C_{ia} C_{jb}
+ *      +m^{ab}g^{ij} g^{kl}m^{cd} C_{ikac} C_{jlbd}
+ * \nonumber\\
+ *      & + & m^{ab}\Lambda^2 C_a C_b
+ *      +m^{ab}\Lambda^2 g^{ij}m^{cd} C_{iac} C_{jbd}.
+ * \f} Here \f$m^{ab}\f$ is an arbitrary positive-definite matrix, and
+ * \f$\Lambda\f$ is an arbitrary real scalar.
+ * Choose \f$m^{ab} = \delta^{ab}\f$ but allow an arbitrary coefficient to be
+ * placed in front of each term. Then, absorb \f$\Lambda^2\f$ into one of
+ * these coefficients, to get
+ * \f{eqnarray}{ E &=& K_
+ * F\delta^{ab} F_a F_b +K_2\delta^{ab}g^{ij} C_{ia} C_{jb}
+ +K_4\delta^{ab}g^{ij}
+ * g^{kl}\delta^{cd} C_{ikac} C_{jlbd}
+ * \nonumber\\
+ *      & + & K_1\delta^{ab} C_a C_b
+ *      +K_3\delta^{ab} g^{ij}\delta^{cd} C_{iac} C_{jbd}.
+ * \f}
+ * Adopting a Euclidean norm for the constraint space (i.e., choosing to raise
+ and
+ * lower spacetime indices with Kronecker deltas) gives
+ * \f{eqnarray}{ E &=& K_ F
+ * F_a F_a +K_2g^{ij} C_{ia} C_{ja} +K_4 g^{ij} g^{kl} C_{ikac} C_{jlac}
+ * \nonumber\\
+ *      & + & K_1 C_a C_a
+ *      +K_3g^{ij} C_{iac} C_{jac}.
+ * \f} The two-index constraint and f constraint can be viewed as the
+ * time and space components of a combined spacetime constraint. So next
+ * choose
+ * \f$K_ F=K_2\f$, giving \f{eqnarray}{ E&=& K_1 C_a C_a + K_2\left(F_a F_a
+ *      + C_{ia} C_{ja} g^{ij}\right)\nonumber\\
+ *      & + & K_3 C_{iab} C_{jab} g^{ij}
+ *      + K_4 C_{ikab} C_{jlab}g^{ij} g^{kl}.
+ * \f}
+ *
+ * Note that \f$C_{ikab}\f$ is antisymmetric on the first two indices. Next,
+ * replace the four-index constraint \f$C_{ijab}\f$ with \f$D_{iab}\f$, which
+ * contains the independent components of \f$C_{ijab}\f$. Specifically,
+ * \f{eqnarray}{
+ * D_{iab} \equiv \frac{1}{2} \epsilon_{i}{}^{jk} C_{jkab}.
+ * \f} The inverse relationship is
+ * \f{eqnarray}{
+ * C_{jkab} = \epsilon^{i}{}_{jk} D_{iab},
+ * \f} where \f$\epsilon_{ijk}\f$ is the flat-space Levi-Civita tensor, whose
+ * indices are raised and lowered with the Kronecker delta. Inserting this
+ relation
+ * to replace \f$C_{jkab}\f$ with \f$D_{iab}\f$ gives \f{eqnarray}{ E &=& K_1
+ C_a
+ * C_a + K_2\left(F_a F_a
+ *      + C_{ia} C_{ja} g^{ij}\right)\nonumber\\
+ *      & + & K_3 C_{iab} C_{jab} g^{ij}\nonumber\\
+ *      & + & K_4 D_{mab} D_{nab} \epsilon^{m}{}_{ik}
+ *      \epsilon^{n}{}_{jl} g^{ij} g^{kl}. \f}
+ *
+ * There's a subtle point here: \f$g^{ij}\f$ is the inverse spatial metric,
+ which
+ * is not necessarily flat. But \f$\epsilon^{i}{}_{jk}\f$ is the flat space
+ * Levi-Civita tensor. In order to raise and lower indices of the Levi-Civita
+ * tensor with the inverse spatial metrics, put in the appropriate factors of
+ * \f$\sqrt{g}\f$, where \f$g\f$ is the metric determinant, to make the
+ * \emph{curved-space} Levi-Civita tensor compatible with \f$g^{ij}\f$. Let
```

Comment:
this `\emph` isn't showing up correctly. I think the markdown equivalent is `*` around the word. If you found the conversion script useful overall then I can have it automate this change as well in the future.